applications/machine_learning/src/modules/Kconfig.led_state

```diff
@@ -11,6 +11,28 @@ menuconfig ML_APP_LED_STATE
 
 if ML_APP_LED_STATE
 
+config ML_APP_LED_STATE_ANOMALY_THRESH
+	int "Anomaly threshold [in 0.001 units]"
+	default 600
+	help
+	  If anomaly value reported by the machine learning model is greater than the defined
+	  anomaly threshold, the result is marked as anomaly and label assigned by the model is
+	  ignored.
+
+config ML_APP_LED_STATE_VALUE_THRESH
+	int "Value threshold [in 0.001 units]"
+	default 800
+	help
+	  If value reported by the machine learning model is smaller than the defined threshold,
+	  the result is ignored.
+
+config ML_APP_LED_STATE_PREDICTION_STREAK_THRESH
+	int "Number of subsequent predictions required to report result"
+	default 2
```

Comment:
so 2 in the end?

Reply:
Actually it's just a default value. I treated it is as "prediction and confirmation".
This is highly dependant on application and it's configuration (e.g. prediction window shift).